server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -226,7 +227,24 @@ private TrTag renderApplicantRow(Account applicant) {
             StyleUtils.even(Styles.BG_GRAY_100))
         .with(renderInfoCell(applicant))
         .with(renderApplicantInfoCell(applicant))
-        .with(renderActionsCell(applicant));
+        .with(renderActionsCell(applicant))
+        .with(renderDateOfBirthCell(applicant));
+  }
+
+  private TdTag renderDateOfBirthCell(Account applicantAccount) {
+    Optional<Applicant> newestApplicant = applicantAccount.newestApplicant();
+    if (newestApplicant.isEmpty()) {
+      return td().withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.PR_12);
+    }
+    String currentDob =
+        newestApplicant
+            .get()
+            .getApplicantData()
+            .getDateOfBirth()
+            .map(LocalDate::toString)
```

Comment:
suggestion (blocking): would prefer a method in DateConverter that specifies the format rather than implicitly relying on the default format of `LocalDate::toString`

Reply:
The whole point of toString() is just this. I don't see a need for another method.

Reply:
`LocalDate::toString` formats the date slightly differently than the one format used throughout the app

Reply:
See example: https://github.com/civiform/civiform/blob/main/server/app/services/applicant/question/DateQuestion.java#L57

It'd be good to move to using `DateConverter` for formatting all dates in the system so that we can be sure they're displayed and parsed consistently.

Reply:
Done